fairlearn/reductions/_moments/conditional_selection_rate.py

```diff
@@ -141,6 +156,12 @@ class DemographicParity(ConditionalSelectionRate):
 
     short_name = "DemographicParity"
 
+    def __init__(self, ratio=1.0):
+        super(DemographicParity, self).__init__()
```

Comment:
Is this common to all subclasses of ConditionalSelectionRate? If so, just move it to the parent class

Reply:
Yes, it is common. Will move this to the parent class.